my-dsl.md

```diff
@@ -1,38 +1,30 @@
 # Language
-_What is the name of the language? Link the name to its webpage 
-(if appropriate)._
+[Protocol Buffers](https://developers.google.com/protocol-buffers/)
 
 
 # Domain
-_Describe the language's domain in five words._
+Sending and processing language-neutral data.
 
 
 # Computational model
-_What is the underlying computational model of this language? To answer this 
-question, provide a high-level description (no more than 100 words) of the 
-computation that occurs when someone executes a program in this language._
+The file that defines the messages is compiled, and then instances of the messages used by other applications are encoded to binary based on the object file.
 
 
 # DSL-ness
-_Fowler writes about a spectrum of languages, from general-purpose languages to 
-"purely" domain-specific. Where does the DSL you chose fall on this spectrum, 
-and why?_ 
+Protocol Buffers falls much closer to the domain-specific side of the spectrum. Fowler often refers to whether a language supports conditional logic and looping to determine whether it's general-purpose, and as far as I know, Protocol Buffers supports neither feature.
 
```

Comment:
Also, it seems like this language is designed for a very specific purpose in mind. That is often how you can tell what a DSL is because sometimes a DSL will have common language features like conditional logic and looping. Just because a language might have these features doesn't mean they were meant to be used.